File: Java/1275-1.java
// Math.pow()를 이용한 다른 풀이
import java.util.Scanner;

public class Main{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        int k = sc.nextInt();
        
        System.out.println((int)Math.pow(n, k));
    }
}

File: C/1275-1.c
// math.h를 이용한 다른 풀이
#include <stdio.h>
#include <math.h>

int main(){
	int n, k;
	scanf("%d %d", &n, &k);
	int res = pow(n, k);
	printf("%d", res);
    return 0;
}